client/gutenberg/editor/edit-post/assets/stylesheets/main.scss

```diff
@@ -152,8 +152,6 @@ body.gutenberg-editor-page {
 	input[type="number"],
 	select,
 	textarea {
-		margin-top: 0; // These override a "margin: 1px" from core.
```

Comment:
Just a heads up: all the changes on this file are coming from having pasted the most recent version of the [`@wordpress/edit-post` styles].(https://github.com/WordPress/gutenberg/blob/9097373012150c50bd11e9b554b79ba0d1fabb69/packages/edit-post/src/style.scss)